packages/drivers/debugger/src/fluidDebuggerController.ts

```diff
@@ -310,3 +337,39 @@ export class DebugReplayController extends ReplayController implements IDebugger
         this.startSeqDeferred.resolve(seq);
     }
 }
+
+async function* generateSequencedMessagesFromDeltaStorage(deltaStorage: IDocumentDeltaStorageService)  {
+    const timeStart = Date.now();
+    let requests = 0;
+    let opsStorage = 0;
+    let lastSeq = 0;
+    const batch = 2000;
+    while (true) {
+        requests++;
+        const messages = await loadChunk(lastSeq, lastSeq + batch, deltaStorage);
+        if (messages.length === 0) {
+            break;
+        }
+        yield messages;
+        opsStorage += messages.length;
+        lastSeq = messages[messages.length - 1].sequenceNumber;
+    }
+
+    if (requests > 0) {
+        // eslint-disable-next-line max-len
+        console.log(`\n${Math.floor((Date.now() - timeStart) / 1000)} seconds to retrieve ${opsStorage} ops in ${requests} requests`);
```

Comment:
I'd remove console logging - more code for not much value (other then initial debugging)

Reply:
Same across whole change.